src/main/resources/templates/add.html

```diff
@@ -1,43 +1,35 @@
 <!DOCTYPE html>
-<html xmlns:th = "http://www.thymeleaf.org">
+<html lang="en" xmlns:th = "http://www.thymeleaf.org">
 <head>
+  <title>Investment Tracker</title>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Investment Tracker</title>
   <link rel="stylesheet" href="style.css">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+  <script src="script.js"></script>
   <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
   <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
   <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
-  <script type="text/javascript">
-    $(function() {
-      $("#company_Symbol").autocomplete({
-        source: "dataAutoComplete",
-        minLength: 1
-      });
-    });
-  </script>
```

Comment:
This was moved to script.js and slightly rewritten. 
https://www.geeksforgeeks.org/what-is-external-javascript/
External JS is incredibly important as it makes your code more legible. Internal JS is fine, but if you're ever planning on expanding this function, it can quickly grow out of control.

Reply:
Agreed.